test_runner/fixtures/neon_fixtures.py

```diff
@@ -2353,18 +2362,9 @@ def workdir(self) -> Path:
         return self.env.repo_dir / f"pageserver_{self.id}"
 
     def assert_no_errors(self):
-        logfile = self.workdir / "pageserver.log"
-        if not logfile.exists():
-            log.warning(f"Skipping log check: {logfile} does not exist")
-            return
-
-        with logfile.open("r") as f:
-            errors = scan_pageserver_log_for_errors(f, self.allowed_errors)
```

Comment:
Did you remove the `scan_pageserver_log_for_errors` method? Maybe I'm not seeing it in the diff.

Reply:
It's still used in the standalone execution mode of `allowed_errors.py`.  I'm asking in slack if that's actually used anywhere...

Reply:
I think it might be used for benchmarks that run in staging. Not sure I see why it can't be removed and replaced with a call to the new method.